### ISSUE TITLE
Link to GitHub for exact commit or tag

### ISSUE THREAD

**Issue by hroncok** [nait-aul/doksit#52]
I've seen at your talk that the documentation links to the source on GitHub and the current branch is used in the link. I'd suggest to use the commit hash or tag (if HEAD happens to be a tag as well), otherwise the links will eventually stop working.

**Comment by nait-aul** [comment]
Yup, btw. the reason why I got the 404 error is that I created a local branch (pycon) and not pushed it to GitHub (there is only for master).

**Comment by hroncok** [comment]
I've figured that out ;)
